mythplugins/mythmusic/mythmusic/visualize.h

```diff
@@ -140,10 +140,6 @@ class MonoScope : public StereoScope
 
 // WaveForm - see whole track - by twitham@sbcglobal.net, 2023/01
 
-#define WF_AUDIO_SIZE 4096     // maximum samples to process at a time
-#define WF_WIDTH 1920   // image cache size, will scale to any display
-#define WF_HEIGHT 1080
-
```

Comment:
These should stay.  You could convert them to something like the following so that the compiler can understand and process them instead of doing a blind text substitution.

static constexpr uint32_t WF_AUDIO_SIZE {4096};
static constexpr int32_t WF_WIDTH {1920};
static constexpr int32_t WF_HEIGHT {1080};

Reply:
WFAudioSize restored as you suggest.  Width and height moved a few lines lower to:

    QSize         m_wfsize {1920, 1080}; // picture size

Are you saying the define or static constexpr is preferred over this?  Even if used only in this definition?  The .cpp then uses m_wfsize.height() and m_wfsize.width(), getting those values from this QSize rather than the former fixed numbers.  Perhaps reverting to static numbers is more efficient than QSize?

Reply:
static const is preferred.  I accept that 1920x1080 is a common enough size that someone reading the code would recognize it, so I won't say anything more about that.

I don't have any problem with using a QSize object.